src/createServer.js

```diff
@@ -1,8 +1,66 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const fileName = normalizedURL.pathname.slice(1);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!req.url.startsWith('/file')) {
+      res.statusCode = 400;
+      res.end('Invalid request');
+
+      return;
+    }
+
+    if (!req.url.startsWith('/file/')) {
+      res.end('Hint message: Use /file/ to access files');
+
+      return;
+    }
+
+    if (req.url.includes('//')) {
+      res.statusCode = 404;
+      res.end('Invalid path');
+
+      return;
+    }
```

Comment:
The check for double slashes in the URL (`if (req.url.includes('//'))`) is a good practice to prevent malformed URLs. Ensure that this check is necessary for your specific use case, as it might not be needed if URLs are normalized correctly.